Dockerfile

```diff
@@ -1,5 +1,5 @@
 # vim: set expandtab sw=4
-FROM golang:1.14-buster
+FROM golang:1.16-buster
```

Comment:
Did you have any issue with `golang-docker` version that was mentioned above?

Reply:
Great question. `golang-docker` that was mentioned above is refering to this [golang-docker repo](https://github.com/GoogleCloudPlatform/golang-docker). That repo [produces](https://github.com/GoogleCloudPlatform/golang-docker/blob/c990cc53befd78a21a641e7ab2c0873ff825a3ed/runtime-builder/cloudbuild.yaml#L5) the image gcr.io/gcp-runtimes/go1-builder:{VERSION}. That image was used in the docker image for [searchcache](https://github.com/web-platform-tests/wpt.fyi/pull/2846/files#diff-c6074bb468b15549341ef7f0888b12cb9176e20446d03c3dc15f006fb3ac2d9eL3-R4) for building the Go executable and is not used for the final image. Kyle mentioned that this would be blocked until a [PR](https://github.com/GoogleCloudPlatform/golang-docker/pull/194) was merged. Instead of waiting for the PR to be merged, I changed the builder image to use the official golang docker image from docker hub.

Reply:
Great work on migrating it off `golang-docker`. Looking at the PR history, I couldn't find the reason why the `gcr.io/gcp-runtimes/go1-builder` image was used, when there is an official golang docker image.